css/edit-media-page.styl

```diff
@@ -0,0 +1,31 @@
+.media-list
+  text-align: center
+
+.media-item
+  border-radius: 5px
+  display: inline-block
+  flex: 1 0 15em
+  margin-bottom: 1em
+  padding: 0.5em 3vw
+  text-align: center
+  vertical-align: middle
+
+  &-container
```

Comment:
Yeah, I was playing with this because it might be useful for BEM-style stuff. Still not crazy about having super verbose classnames in the markup though.